t/Helper.pm

```diff
@@ -38,6 +38,22 @@ sub validate_ok {
     or Test::More::diag(encode_json(\@errors));
 }
 
+sub validate_custom_format_ok {
```

Comment:
I don't really want this function here, since it's not a generic function that will be used in more test files.